src/Sylius/Behat/Context/Setup/CatalogPromotionContext.php

```diff
@@ -114,6 +115,33 @@ public function itWillReducePrice(CatalogPromotionInterface $catalogPromotion, f
         $this->entityManager->flush();
     }
 
+    /**
+     * @Given /^there is a catalog promotion with "([^"]*)" name that will be applied on ("[^"]+" variant) and will reduce price by ("[^"]+")$/
+     */
+    public function thereIsACatalogPromotionWithNameThatWillBeAppliedOnVariantAndWillReducePriceBy(
+        string $name,
+        ProductVariantInterface $variant,
+        float $discount
+    ): void {
+        $catalogPromotion = $this->createCatalogPromotion($name);
+
+        /** @var CatalogPromotionRuleInterface $catalogPromotionRule */
+        $catalogPromotionRule = $this->catalogPromotionRuleFactory->createNew();
+        $catalogPromotionRule->setType(CatalogPromotionRuleInterface::TYPE_FOR_VARIANTS);
+        $catalogPromotionRule->setConfiguration([$variant->getCode()]);
```

Comment:
i think we can wait with this one, untill https://github.com/Sylius/Sylius/pull/13066 is merged, so we can use 'variants' => [$variant->getCode()]